### PR TITLE
Renamed symbolic names for reosurces to adhere to naming conventions

### PR DESCRIPTION
**Pull request by carlzxc71**
- Renamed symbolic names to be singular or contain underscore
- Added `outputs.tf`
- Fixed spacing
- Locked in TLS1_2 for storage account from tfsec recommendation

### CODE REVIEW

File: deploy/main.tf [hunk 1]
```diff
@@ -1,18 +1,21 @@
-// Resource group configuration
+data "local_file" "script" {
+  filename = "../src/List-PastAlerts.ps1"
+}
+
+data "azurerm_subscription" "primary" {
+}
 
-resource "azurerm_resource_group" "rg-monitor-automation" {
+resource "azurerm_resource_group" "rg" {
```
Comment (simongottschlag) [inline deploy/main.tf]: If you only have one instance of a resource or data source, the convention is usually to name it `this`:

> Resource name should be named this if there is no more descriptive and general name available, or if the resource module creates a single resource of this type (eg, in AWS VPC module there is a single resource of type aws_nat_gateway and multiple resources of typeaws_route_table, so aws_nat_gateway should be named this and aws_route_table should have more descriptive names - like private, public, database).

https://www.terraform-best-practices.com/naming#resource-and-data-source-arguments

File: deploy/main.tf [hunk 2]
```diff
@@ -1,18 +1,21 @@
-// Resource group configuration
+data "local_file" "script" {
+  filename = "../src/List-PastAlerts.ps1"
```
Comment (simongottschlag) [inline deploy/main.tf]: I would have named this `list_past_alerts_script`

File: deploy/main.tf [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
+}
+
+data "azurerm_subscription" "primary" {
+}
```
Comment (simongottschlag) [inline deploy/main.tf]: `primary`, `current` or `this` would be fine with the name here. Make it one line and end it with `{}`

File: deploy/main.tf [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
   name     = var.rg_name
   location = var.rg_location
 
   tags = local.tags
 }
 
-## Automation account configuration
-
-resource "azurerm_automation_account" "aa-monitor-automation" {
+resource "azurerm_automation_account" "account" {
```
Comment (simongottschlag) [inline deploy/main.tf]: > Do not repeat resource type in resource name (not partially, nor completely)

Either `this` or the purpose the resource has. Do not include `automation` or `aa` etc

**Review by simongottschlag — COMMENTED** [review]
You wouldn't name your dog `dog`: https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat

Left a few comments, but looks a lot better. Most of the resources that you only use one instance of can be named `this`.

Keep up the good work!

File: deploy/main.tf (hunk 1 above)
Comment (carlzxc71) [reply deploy/main.tf]: Point taken, sir! :)

File: deploy/main.tf (hunk 2 above)
Comment (carlzxc71) [reply deploy/main.tf]: Updated

File: deploy/main.tf (hunk 4 above)
Comment (carlzxc71) [reply deploy/main.tf]: Fixed

File: deploy/main.tf (hunk 3 above)
Comment (carlzxc71) [reply deploy/main.tf]: Fixed, makes sense

**Pull request merged**